tools/clang/tools/dxildll/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ target_link_libraries(dxildll PRIVATE
   dxcvalidator
 )
 
-if (WIN32)
+if (WIN32 AND HLSL_BUILD_DXILCONV)
```

Comment:
We currently disable `HLSL_BUILD_DXILCONV` in our build as we don't need it, and it speeds up our builds at Google. I would rather we keep this guard.

Reply:
Only DxcRuntimeEtw is guard by HLSL_BUILD_DXILCONV currently because dxcetw is used by dxcompiler.
But dxildll does use DxcRuntimeEtw, without it dxildll will get error for cannot find DxcRuntimeEtw.h.

Maybe we should add HLSL_BUILD_DXILDLL, and guard DxcRuntimeEtw with (HLSL_BUILD_DXILCONV or HLSL_BUILD_DXILDLL)?

Reply:
Oops, I pushed too early. Still fixing this.

Reply:
Okay, I pushed what I think you asked for. However, I'm not sure if adding `HLSL_BUILD_DXILDLL` is what we want. This is perhaps a maintenance burden that isn't warranted for DXC.

Reply:
If we always build dxildll, then there's no need to guard DxcRuntimeEtw since it will be required by dxildll.
That will be another way to fix this issue.
Let's see how others feel about it.

Reply:
Yes, exactly what I was thinking! I just pushed https://github.com/microsoft/DirectXShaderCompiler/pull/6912 which does exactly that.